OpenSim/Region/Framework/Scenes/Scene.Inventory.cs

```diff
@@ -4247,7 +4247,14 @@ private bool CheckGreyGoo(SceneObjectPart sourcePart, SceneObjectGroup newGroup)
                         reason = "success";
                         return allGroups;
                     }
+                } else
```

Comment:
(nitpick) Isn't it normal in this codebase to drop the else onto it's own line or am I missing something?